README.md

```diff
@@ -1,46 +1,44 @@
-______________________________________________________  
-
 # Hang in There  
 
 ### Abstract:
 [//]: <> (Briefly describe what you built and its features. What problem is the app solving? How does this application solve that problem?)
+We built a website that allows you to generate random motivational posters as well as allowing users to create their own motivational poster. There is also a feature where the user can save posters that they wish to keep for future reference. They also have the ability to delete posters as they wish. This gives the user the capability to continuously change their motivation resources.
 
 ### Installation Instructions:
 [//]: <> (What steps does a person have to take to get your app cloned down and running?)
+1. Click the green <> Code button. Confirm SSH is selected. Click the Copy Code button at the end of the link (the icon looks like a box on top of a box). 
+2. Go into your terminal. After confirming you are in the right location, type "git clone [insert link copied from previous step]". Hit Return/Enter.
+3. CD into the cloned file. Type "code ." to open up files in your code viewer (i.e., VS Code).
+4. Enter "open index.html" into your console in order to open the website. This is where you will see your local changes.
 
 ### Preview of App:
 [//]: <> (Provide ONE gif or screenshot of your application - choose the "coolest" piece of functionality to show off.)
+![Overview .gif](https://media.giphy.com/media/MwJltY6DOHmwhrwM5V/giphy.gif)
 
 ### Context:
-[//]: <> (Give some context for the project here. How long did you have to work on it? How far into the Turing program are you?)
+[//]: <> (Give some context for the project here. How long did you have to work on it? How far into the Turing program are you?) 
+We were allotted 5.5/6.0 days for this project. We are two weeks into the Frontend program. 
 
 ### Contributors:
 [//]: <> (Who worked on this application? Link to their GitHubs.)
-Rob Schlereth/Rachel Brendel PM: Kayla Gordon
+Rachel Brendel [https://github.com/brendel-r] <br> 
+Rob Schlereth [https://github.com/rschlereth] <br>
+PM: Kayla Gordon
+
 ### Learning Goals:
 [//]: <> (What were the learning goals of this project? What tech did you work with?)
+Learning not only to use JavaScript to manipulate data from HTML and CSS, but also how to understand the connection between JavaScript, HTML, and CSS. Using classes and instances. Learning how to manipulate the page after it's been loaded by adding and deleting elements on the DOM. Building an understanding of writing code with someone else/a partner. Communicate, troubleshoot, and plan as a team effort. Use proper Git workflow in our processes. We utilized all of our resources to successfully accomplish our end goal as a team. 
 
 ### Wins + Challenges:
 [//]: <> (What are 2-3 wins you have from this project? What were some challenges you faced - and how did you get over them?)
+A few wins for us during this project:
+
```

Comment:
Proud of us!!! even though I feel like we need more practice, We were able to really get some major wins in the end!!!